packages/apollo-forest-run/src/forest/types.ts

```diff
@@ -57,6 +60,18 @@ export type IndexedForest = {
 export type Source = Readonly<SourceObject | SourceCompositeList>;
 export type Draft = SourceObject | SourceCompositeList;
 
+export type UpdateChunkMetadata = {
```

Comment:
Let's stick with more specific "Stats" suffix here vs "Metadata"

```suggestion
export type UpdateChunkStats = {
```